index.html

```diff
@@ -87,6 +87,17 @@
           </div>
         </div>
         <h1 class="heading-48">Component Library</h1>
+        <div class="alert alert--info">
+          <p class="alert__message">
+            This component library is a visual representation of the styles used to build HMRC's services
+          </p>
+          <h3>Questions and Issues</h3>
+          <p class="alert__message">
+          If you have questions about how to use either a component or the library, or would like to
+            <a href="http://hmrc.github.io/assets-frontend#reporting-issues--actioning-work">report an issue</a> then please get in touch using one of our
+            <a href="http://hmrc.github.io/assets-frontend#support-channels">support channels</a>.
```

Comment:
We should probably group all the support options under a top level header and then link to that